src/main/kotlin/mate/academy/Main.kt

```diff
@@ -1,7 +1,9 @@
 package mate.academy
 
 fun greetUsers() {
-    // implement function
+    println("Please enter a name (or press Enter to quit):")
+    val name = readLine()
```

Comment:
The `readLine()` function can return `null` if the user presses Enter without typing anything. Consider adding a check to handle this case, such as providing a default name or prompting the user again.